sim/ExternalRepos.mk

```diff
@@ -47,3 +47,8 @@ DPI_DASM_SPIKE_HASH   ?= 8faa928819fb551325e76b463fc0c978e22f5be3
 SVLIB_REPO       ?= https://bitbucket.org/verilab/svlib/src/master/svlib
 SVLIB_BRANCH     ?= master
 SVLIB_HASH       ?= c25509a7e54a880fe8f58f3daa2f891d6ecf6428
+
+#riscv-arch-test suite with riscof
+RISCOF_ARCH_TEST_SUITE_REPO    ?= https://github.com/riscv/riscv-arch-test.git
+RISCOF_ARCH_TEST_SUITE_BRANCH  ?= main
+RISCOF_ARCH_TEST_SUITE_TAG     ?= latest
```

Comment:
"latest" might be good for now, but I suggest you select the hash of the current head of the repo.  That repo is outside of OpenHW control and it may change in ways that are not compatible with this repo.

Reply:
Good point, I added version 3.8.9 as the tag as this is the latest version currently.